doc/quickstart/52nWPS_quickstart.rst

```diff
@@ -18,53 +19,53 @@ This quickstart describes how to process data and store the result in another we
 
 .. contents:: Contents
 
-Running
```

Comment:
Not sure about this one, this text seems to be replaced for the translations, see here: https://github.com/OSGeo/OSGeoLive-doc/blob/master/locale/de/LC_MESSAGES/quickstart/52nWPS_quickstart.po#L48

Reply:
Oh! good point. I'll revert that change. I'd rather not mess with the translations.

Reply:
No I'm going to leave it in. I went back through the IRC logs and found where we talked about it. The team agreed at the start of my project that I'll do English and then the translators will take these from git to work on. This is for version 14 so it should get picked up for that. @bpross-52n Are you okay with leaving this change in?

Reply:
Yes, it is ok. Just wanted to make sure. Thanks for checking!